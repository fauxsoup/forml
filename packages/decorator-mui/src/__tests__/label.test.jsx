import Label from "../label";
import Context from "@forml/context";
import React from "react";
import { render } from "@testing-library/react";
import * as decorator from "../";

describe("renders", function() {
    let form;
    let title = "title";
    let description = "description";

    beforeEach(function() {
        form = { type: "string" };
    });

    test("with no shrink", function() {
        const { container } = render(
            <Context.Provider value={{ decorator }}>
                <Label />
            </Context.Provider>
        );

        expect(container).toMatchSnapshot();
    });

    test("with shrink", function() {
        const { container } = render(
            <Context.Provider value={{ decorator }}>
                <Label />
            </Context.Provider>
        );

        expect(container).toMatchSnapshot();
    });
});
