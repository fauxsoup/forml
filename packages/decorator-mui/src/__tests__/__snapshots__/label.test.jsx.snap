// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with no shrink 1`] = `
<div>
  <label
    class="MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-root MuiFormLabel-colorPrimary css-9npbnl-MuiFormLabel-root-MuiInputLabel-root"
    data-shrink="false"
  />
</div>
`;

exports[`renders with shrink 1`] = `
<div>
  <label
    class="MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-root MuiFormLabel-colorPrimary css-9npbnl-MuiFormLabel-root-MuiInputLabel-root"
    data-shrink="false"
  />
</div>
`;
